### PR TITLE
Update top level logging documentation

### PR DESCRIPTION
**Pull request by satnam6502**

### CODE REVIEW

**Comment by mbforbes** [comment]
(Pressed for time—@satnam6502 assigning to you to find a reviewer. Someone should take a one-over, at least. My one questions is why we're removing all mentions of elasticsearch [perhaps it's all on subpages now?])

**Comment by satnam6502** [comment]
A different page has been added for Elasticsearch + Kibana and I will link to that in a subsequent PR.

**Comment by satnam6502** [comment]
Added a section about Elasticsearch and Kibana.

**Comment by satnam6502** [comment]
Removed absolute paths at the request of @RichieEscarez

**Comment by a-robinson** [comment]
Looks great, sorry if I gave you outdated advice about using absolute rather than relative paths.

File: docs/logging.md [hunk 1]
```diff
@@ -3,47 +3,69 @@
...
-To learn how to query, filter etc. using Kibana you might like to look at this [tutorial](http://www.elasticsearch.org/guide/en/kibana/current/working-with-queries-and-filters.html).
-
-You can check to see if any logs are being ingested into Elasticsearch by curling against its URL. You will need to provide the username and password that was generated when your cluster was created. This can be found in the `kubernetes_auth` file for your cluster.
+## Examining the logs of running containers
+The logs of a running container may be fetched using the command `kubectl logs`. For example, given
+this pod specification which has a container which writes out some text to standard
+output every second [counter-pod.yaml](/examples/blog-logging/counter-pod.yaml):
 ```
-$ curl -k -u admin:Drt3KdRGnoQL6TQM https://130.211.152.93/api/v1/proxy/namespaces/default/services/elasticsearch-logging/_search?size=10
+ apiVersion: v1
```
Comment (mbforbes) [inline docs/logging.md]: probably not worth doing a re-push for this, but I (personally) like having syntax highlighting for supported languages in documentation, so in the future could you toss a \`\`\`\<lang> at the top (in this case \`\`\`yaml)?

**Pull request merged**